frontend-react/src/layouts/MainLayout.tsx

```diff
@@ -0,0 +1,43 @@
+import { GovBanner } from "@trussworks/react-uswds";
+import classNames from "classnames";
+import { Outlet, useMatches } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+
+import App from "../App";
+import { DAPHeader } from "../components/header/DAPHeader";
+import { ReportStreamHeader } from "../components/header/ReportStreamHeader";
+import { ReportStreamFooter } from "../components/ReportStreamFooter";
+import SenderModeBanner from "../components/SenderModeBanner";
+import { USLink } from "../components/USLink";
+import { useSessionContext } from "../contexts/SessionContext";
+
+const MainLayout = () => {
+    const { environment } = useSessionContext();
+    const matches = useMatches();
+    const { handle = {} } = matches.at(-1) ?? {};
+    const { isContentPage } = handle as any;
+
+    return (
+        <App>
+            <DAPHeader env={environment} />
+            <USLink className="usa-skipnav" href="#main-content">
+                Skip Nav
+            </USLink>
+            <GovBanner aria-label="Official government website" />
+            <SenderModeBanner />
+            <ReportStreamHeader />
+            <main
+                id="main-content"
+                className={classNames(isContentPage && "rs-style__content")}
```

Comment:
Maybe just a comment on what `isContentPage` is used for (either here or in the `AppRouter`) for future developers?

Reply:
Will do.

Reply:
JsDoc comment added to property definition that will be visible from your IDE intellisense.